tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
tab spacing is weird in here

Reply:
another instance of my editor hiding stuff form my apparently. Looks like tabs were being used for tabs instead of spaces. I hope I got it fixed but i guess I won't know until I push up 😬